gax/src/main/java/com/google/api/gax/retrying/BasicResultRetryAlgorithm.java

```diff
@@ -51,6 +54,23 @@ public TimedAttemptSettings createNextAttempt(
     return null;
   }
 
+  /**
+   * Always returns null, indicating that this algorithm does not provide any specific settings for
+   * the next attempt.
+   *
+   * @param prevThrowable exception thrown by the previous attempt ({@code null}, if none)
+   * @param prevResponse response returned by the previous attempt
+   * @param prevSettings previous attempt settings
+   */
+  @Override
+  public TimedAttemptSettings createNextAttempt(
+      RetryingContext context,
```

Comment:
prev --> previous
Google style does not all abbreviations in variable names

Reply:
(This one was already done, but not marked as such)